proto/v1/database_service.proto

```diff
@@ -394,4 +446,60 @@ message BackupSetting {
 
   // hook_url(https://www.bytebase.com/docs/administration/webhook-integration/database-webhook) is the URL to send a notification when a backup is created.
   string hook_url = 4;
-}
\ No newline at end of file
+}
+
+// OperationMetadata is the metadata for database operations.
+// Empty for now.
+message OperationMetadata {}
```

Comment:
Let's not introduce this for now?

Reply:
Follow the [AIP-151](https://google.aip.dev/151):
> The metadata type is used to provide information such as progress, partial failures, and similar information on each GetOperation call. The metadata type should not be google.protobuf.Empty, unless it is certain that metadata will never be needed. **If metadata might be added in the future, define an empty message for the RPC metadata and use that.**